lib/ssl/src/ssl_handshake.erl

```diff
@@ -775,7 +775,8 @@ decode_suites('3_bytes', Dec) ->
 
 available_suites(UserSuites, Version) ->
     lists:filtermap(fun(Suite) ->
-			    lists:member(Suite, ssl_cipher:all_suites(Version))
+			    lists:member(Suite, ssl_cipher:all_suites(Version) ++
```

Comment:
I realized last night it would be more efficient to concatenate the lists outside of filtermap/2, to avoid building it over and over

Reply:
Good idea :) We have not reviewed it yet, I just included it for first round of testing.